src/LobbyForm.tsx

```diff
@@ -79,26 +109,36 @@ const hostStyles = StyleSheet.create({
     fontSize: "8px",
   },
   button: {
+    padding: "0px",
+    boxSizing: "border-box",
     height: "100%",
     border: "none",
-    borderRadius: "0 8px 8px 0",
+    borderRadius: "60px",
     minWidth: "32px",
-    backgroundColor: "#EEEEEE",
+    backgroundColor: "#1D1E33",
+    ":hover": {
+      backgroundColor: "#069AB4",
+    },
+    ":active": {
+      backgroundColor: "#9CF6FC",
+    },
+  },
+  button2: {
     ":hover": {
-      backgroundColor: "#DDDDDD",
+      backgroundColor: "#7139E0",
     },
     ":active": {
-      backgroundColor: "#CCCCCC",
+      backgroundColor: "#BA9CFC",

```

Comment:
Let's use a better name than "button2", something like "actionButton" or like "joinButton" or something would be better than "button2"

Reasoning: 
If I'm new to the codebase, it's a lot easier to read "joinButton" rather than "button2" since "joinButton" gives context that it's for the "join" feature.